.travis.yml

```diff
@@ -19,25 +19,31 @@ matrix:
     - {python: 3.5, env: TOXENV=py35-django19-nomigrations}
     - {python: 3.5, env: TOXENV=py35-django19}
     - {python: 3.6, env: TOXENV=py36-django19-nomigrations}
-    #- {python: 3.5, env: TOXENV=py35-nomigrations-nocoverage-browser-admin}
-    #- {python: 3.5, env: TOXENV=py35-nomigrations-nocoverage-browser-front}
+    - {python: 2.7, env: TOXENV=py27-django111-nomigrations}
+    - {python: 3.5, env: TOXENV=py35-django111-nomigrations}
+    - {python: 3.5, env: TOXENV=py35-django111}
+    - {python: 3.6, env: TOXENV=py36-django111-nomigrations}
+    - {python: 3.5, env: TOXENV=py35-django19-nomigrations-nocoverage-browser-admin}
+    - {python: 3.5, env: TOXENV=py35-django19-nomigrations-nocoverage-browser-front}
+    - {python: 3.5, env: TOXENV=py35-django111-nomigrations-nocoverage-browser-admin}
+    - {python: 3.5, env: TOXENV=py35-django111-nomigrations-nocoverage-browser-front}
 install:
   - pip install -U pip
   - pip install tox
   - pip install coveralls
   - pip install codecov
 before_install:
-  #- wget https://github.com/mozilla/geckodriver/releases/download/v0.19.1/geckodriver-v0.19.1-linux64.tar.gz
-  #- mkdir geckodriver
-  #- tar -xzf geckodriver*.tar.gz -C geckodriver
-  #- export PATH=$PATH:$PWD/geckodriver
-  #- "nvm install 6"
-  #- "nvm use 6"
-  #- "export DISPLAY=:99.0"
-  #- "sh -e /etc/init.d/xvfb start"
-  #- "npm install -g selenium-standalone@4.5.3"
-  #- "selenium-standalone install"
-  #- "selenium-standalone start &"
+  - wget https://github.com/mozilla/geckodriver/releases/download/v0.20.1/geckodriver-v0.20.1-linux64.tar.gz
+  - mkdir geckodriver
+  - tar -xzf geckodriver*.tar.gz -C geckodriver
+  - export PATH=$PATH:$PWD/geckodriver
+  - "nvm install 6"
+  - "nvm use 6"
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - "npm install -g selenium-standalone@4.9.1"
+  - "selenium-standalone install"
+  - "selenium-standalone start &"
```

Comment:
This should probably moved to a separate script file, like `travis_install_selenium` and the command here could then be just `./travis_install_selenium`.  Running of those should also be done only for the envs that need the selenium stuff, e.g. the listed 4 browser test envs. You can have before_install on per env setting in the Travis build matrix to achieve that:
```yaml
matrix:
  include:
    - ...
    - {python: 3.6, env: TOXENV=py36-django111-nomigrations}
    - python: "3.5"
      env: TOXENV=py35-django19-nomigrations-nocoverage-browser-admin
      before_install: ./travis_install_selenium
    - ...
```